options.lua

```diff
@@ -4,8 +4,9 @@ local RegEvent = MySlot.regevent
 
 
 local f = CreateFrame("Frame", nil, UIParent)
-f.name = L["Myslot"]
-InterfaceOptions_AddCategory(f)
+local category, layout = Settings.RegisterCanvasLayoutCategory(f, L["Myslot"]);

```

Comment:
layout unused?

then why not 

```suggestion
Settings.RegisterAddOnCategory(Settings.RegisterCanvasLayoutCategory(f, L["Myslot"]))
```